config/production/environmentConfiguration.gs

```diff
@@ -8,7 +8,7 @@ var environmentConfiguration = {
   "debugEnabled": false,
   "features": {
     "updateJira": {
-      "enabled": false
+      "enabled": true
```

Comment:
@ljay79  - this will enable the menu item in production now.

Reply:
As mentioned on other place; for the record:

Why would we do this custom enabling per feature param instead of enabling it like any other menu item?
I dont see any benefit or purpose for such.
It either live in production or it isnt.

Reply:
Useful to have feature switches to turn on and off new functionality - agreed where this only controls one menu option its not as valuable - fair point.

Reply:
Useful for development env only - if ever. Once a feature is going into production, a switch like this is useless for this particular feature. That what i mean.
Do i miss a point here?

Reply:
I think its useful to be able to quickly turn functionality off. Features switches work best though when you can change them without releasing (which this wouldnt work with). I've removed the code for that feature switch

Reply:
Thx